README.md

```diff
@@ -154,12 +154,8 @@ def run_experiment(C=1.0, gamma=0.7):
     )
     return score
 
-dispatch_id =
-    run_experiment.dispatch(
-    	C=1.0,
-    	gamma=0.7
-    )
-result = ct.get_result(dispatch_id)
+dispatch_id = run_experiment.dispatch(C=1.0, gamma=0.7)
```

Comment:
I saw the rendered version on your branch and it seems that someone will have to scroll horizontally to see the whole line. So, what we currently have looks like this:
```python
dispatch_id =
    run_experiment.dispatch(
    	C=1.0,
    	gamma=0.7
    )
```
What you have is this:
```python
dispatch_id = run_experiment.dispatch(C=1.0, gamma=0.7)
```
which is perfectly fine if not for the side scrolling thing.

What do you think about doing it like this?:
```python
dispatch_id = run_experiment.dispatch(
    C=1.0,
    gamma=0.7
)
```
So that it fits without side scrolling?